src/components/common/applicant-info-1.vue

```diff
@@ -461,6 +462,7 @@ export default class ApplicantInfo1 extends Mixins(ActionMixin) {
   }
 
   set actingOnOwnBehalf (value) {
+    if (value) this.setFolioNumber('') // no folio number in case it is on own behalf
```

Comment:
Are you sure about this?

I think someone creating their own NR might still want to enter something in the FN field...?

Reply:
This is because it should send a Folio Number only in case it is on someone else's behalf. Otherwise someone can add a Folio Number, come back and change to their own behalf. In this case the old value would be kept and and sent as Folio Number which is not correct.

Reply:
OK, I see this clears the value in case the user changes from someone else's behalf to their own behalf...

But why not allow a user to enter a FN on their own behalf?

Reply:
I see that there is discussion on FN for premium users, but that doesn't mean their won't create a NR on their own behalf.

Maybe I don't understand. But if the BA/PO say it is so, then ok.

Reply:
> But why not allow a user to enter a FN on their own behalf?

This is a requirement:
Scenario 1: Display folio field
GIVEN I am any type of user
WHEN I un-check "I am submitting this on my own behalf"
THEN I can enter a folio number

Reply:
OK!